opsgenie/resource_opsgenie_team_routing_rule.go

```diff
@@ -43,10 +43,6 @@ func resourceOpsGenieTeamRoutingRule() *schema.Resource {
 				Type:     schema.TypeInt,
 				Required: true,
 			},
-			"timezone": {
-				Type:     schema.TypeString,
-				Required: true,
```

Comment:
we need to change required to optional. Because like the error msg says we cannot set timezone for default rule